cloud_info_provider/core.py

```diff
@@ -203,9 +55,11 @@ def parse_opts():
         help=('Path to the directory containing the needed templates'))
 
     parser.add_argument(
-        '--template-extension',
-        default='ldif',
-        help=('Extension to use for the templates'))
+        'format',
```

Comment:
why not `--format`? Looking for opinions I have mixed feelings about this

Reply:
@gwarf any opinion?

Reply:
So this night I realised that for me it would make more sense to have the cloud provider (ie. openstack, opennebularocci,...) as positional argument than the format. Format is just an output option (important, for sure), and the provider is really at the core of the tool.
So I would more be in favor of no positional arg or using the provider as positional arg.

Reply:
so then let's take the middle road and put everything as option :) 
@orviz could you introduce the change?

Reply:
@enolfc @gwarf It's done; I took the chance to move `--middleware` and `--format` at the top of the displayed options (in the help message), so that they stand out. The main description was also changed to remove the 'BDII' word

Reply:
Perfect, thanks!